src/pages/QuestionPage/index.jsx

```diff
@@ -0,0 +1,144 @@
+import React, { useEffect, useState } from "react";
+import { useLocation } from "react-router-dom";
+import Layout from "../../components/Layout";
+import Loader from "../../components/Loader";
+import NotAvailable from "../../components/NotAvailable";
+import { useQuestion } from "../../context/QuestionContext";
+import { useQuiz } from "../../context/QuizContext";
+
+const QuestionPage = () => {
+  const { questions, fetchQuestions } = useQuestion();
+  const { activeQuiz, setActiveQuiz, clearQuizInfo, fetchQuizInfo, quizInfo } =
+    useQuiz();
+  const location = useLocation();
+  let pathName = location.pathname.split("/");
+  let quizId = pathName[pathName.length - 2];
+  const [activeQuestionNo, setActiveQuestionNo] = useState(0);
+  const [optionStateColor, setOptionColor] = useState("light");
+  const [score, setScore] = useState(0);
+  const [activeOption, setActiveOption] = useState(-1);
+  const [timer, setTimer] = useState();
+  const [timeLeft, setTimeLeft] = useState(15);
+
+  useEffect(() => {
+    clearInterval(timer);
```

Comment:
Instead of using **useState** to store the setInterval timerId, use **useRef**

Reply:
Thanks for pointing out as I am having
error with this useState approach